Cargo.toml

```diff
@@ -1,13 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-#=======================================================================================================================
-# Package
-#=======================================================================================================================
-
 [package]
 name = "demikernel"
-version = "1.5.13"
+version = "0.0.1"
```

Comment:
We should leave this number as is because it has meaning for our Github repo.

Reply:
Ok we can start from these numbers on crates.io. What are the GitHub version numbers used for?

Reply:
We have tags on Github for different releases so that people can build against a specific release if they want to.

Reply:
Done.